PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -558,6 +564,16 @@ def miniAOD_customizeMC(process):
     process.patJets.embedGenPartonMatch = False
     #also jet flavour must be switched
     process.patJetFlavourAssociation.rParam = 0.4
+    
+    from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+    from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process.slimmedGenJetsAK8, cut = 'pt>9999', nLoose = 0)
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process.slimmedGenJetsAK8SoftDropSubJets, cut = 'pt>9999', nLoose = 0)
```

Comment:
```suggestion
```
looks like a duplicate of a change in PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py